### PR TITLE
Improve routing

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

Add xml routing o be consider in other sonata bundles routing.

I am targeting this branch, because change respect BC.

## Changelog

```markdown
### Added
- Added routing in xml: `src\Resources\config\routing\audit.xml`
```

### CODE REVIEW

**Comment by wbloszyk** [comment]
@OskarStark @VincentLanglet 
Can we allow adding defaults and requiurements in path for xml files?

**Comment by phansys** [comment]
For semantic reasons, I think we should keep the these XML nodes separated. Configuration in reusable bundles should be as expressive as possible.

**Review by VincentLanglet — APPROVED** [review]

File: src/Resources/config/routing.yml [hunk 1]
```diff
@@ -1,30 +1,19 @@
 simple_things_entity_audit_home:
-    path: /{page}
+    path: /{page<\d+>?1}
```
Comment (phansys) [inline src/Resources/config/routing.yml:2]: IMO, these changes should be reverted.

Comment (wbloszyk) [reply src/Resources/config/routing.yml:2]: We do not have to but I should add next_maroj note about remove this file.

Comment (wbloszyk) [reply src/Resources/config/routing.yml]: Reverted. Also I add NEXT_MAJOR to remove this file.

File: src/Resources/config/routing.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# NEXT_MAJOR: remove this file and add upgrade note
```
Comment (phansys) [inline src/Resources/config/routing.yml:1]: Shouldn't we update the documentation at [`README.md`](https://github.com/sonata-project/EntityAuditBundle/tree/d68de63687b49d4859ae233ab90bc40e7b089add#appconfigroutingyml) in order to use the routes from the XML file?

Comment (wbloszyk) [reply src/Resources/config/routing.yml:1]: Done.

**Review by VincentLanglet — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**